accepted/calc-functions.md

```diff
@@ -84,6 +83,10 @@ equivalent of `-10%` since percentages are resolved before calculations. To
 handle this, we'll deprecate the global `abs()` function with a percentage and
 recommend users explicitly write `math.abs()` or `abs(#{})` instead.
 
+This also expands calculation parsing to allow constructs like `calc(1
+var(--plus-two))` which are valid CSS but weren't supported by the old
```

Comment:
Nit: for clarity, please mention what `var(--plus-two)` means, something like _"where `--plus-two` has the value `+ 2`"_